cro-clib/src/bindings.rs

```diff
@@ -0,0 +1,260 @@
+/* automatically generated by rust-bindgen */
```

Comment:
Maybe don't commit this file since it's generated on the fly?

Reply:
that's correct, i'll remove it.